apps/new-dealer/index.js

```diff
@@ -511,15 +547,23 @@ module.exports = {
       fields: [
         'contact-address-manual'
       ],
-      next: '/summary',
+      next: '/confirm',
       prereqs: ['/contact-postcode', '/contact-details'],
       backLink: 'contact-postcode',
       locals: {
-        field: 'contact'
+        field: 'contact',
+        section: 'contacts-details'
       }
     },
-    '/summary': {
-      clearSession: false
+    '/confirm': {
+      controller: require('./controllers/confirm'),
+      fields: [
+        'declaration'
+      ],
+      fieldsConfig: require('./fields'),
+      next: '/confirmation'
```

Comment:
you'll need email config here too, guess that will be a different ticket though?